transformation-scripts/semopenalex-subfields.py

```diff
@@ -0,0 +1,303 @@
+                    if not topic_description is None:
+                        topic_description = clean(topic_description)
+                        subfields_graph.add((subfield_uri, SKOS.note, Literal(topic_description,datatype=XSD.string)))
+
+                    # field
+                    topic_field = json_data['field']['id']
+                    if not topic_field is None:
+                        topic_field_id = topic_field.replace("https://openalex.org/", "")
+                        topic_field_uri = URIRef(soa_namespace_fields + topic_field_id)
+                        subfields_graph.add((subfield_uri, SKOS.broader, topic_field_uri))
+
+                    # domain
+
+                    # topics
+
+                    # ids (wikidata, wikipedia)
+                    subfield_wikidata = json_data.get('ids').get('wikidata')
+                    if not subfield_wikidata is None:
+                        subfield_wikidata = clean_url(subfield_wikidata)
+                        subfields_graph.add((subfield_uri, OWL.sameAs, URIRef(subfield_wikidata)))
+
+                    topic_wikipedia = json_data.get('ids').get('wikipedia')
+                    if not topic_wikipedia is None:
+                        topic_wikipedia = clean_url(topic_wikipedia)
+                        subfields_graph.add((subfield_uri, RDFS.seeAlso, URIRef(topic_wikipedia)))
+
+
+                    # updated_date
+                    subfield_updated_date = json_data['updated_date']
+                    if not subfield_updated_date is None:
+                        subfield_updated_date = clean_date(subfield_updated_date)
+                        subfields_graph.add((subfield_uri, DCTERMS.modified, Literal(subfield_updated_date,datatype=XSD.date)))
+
+                    # created_date
+                    subfield_created_date = json_data['created_date']
+                    if not subfield_created_date is None:
+                        subfields_graph.add((subfield_uri, DCTERMS.created, Literal(subfield_created_date,datatype=XSD.date)))
+
+                    # works_count
+                    subfield_works_count = json_data['works_count']
+                    if not subfield_works_count is None:
+                        subfields_graph.add((subfield_uri, works_count_predicate, Literal(subfield_works_count,datatype=XSD.integer)))
+
+                    # cited_by_count
+                    subfield_cited_by_count = json_data['cited_by_count']
+                    if not subfield_cited_by_count is None:
+                        subfields_graph.add((subfield_uri, cited_by_count_predicate, Literal(subfield_cited_by_count,datatype=XSD.integer)))
+
+                    # siblings 
+                    subfield_siblings = json_data['siblings']
+                    if not subfield_siblings is None:
+                        for sibling in subfield_siblings:
+                            sibling_id = sibling['id'].replace("https://openalex.org/", "")
```

Comment:
this is not also correct. 
```suggestion
                            sibling_id = sibling['id'].replace("https://openalex.org/subfields/", "")
```